scenario/context/CometContext.ts

```diff
@@ -407,6 +407,7 @@ async function forkContext(c: CometContext, w: World): Promise<CometContext> {
 export const constraints = [
   new FilterConstraint(),
   new NativeTokenConstraint(),
+  new ProposalConstraint(), // XXX
```

Comment:
Note: remove this after ENS proposal passes, want to assume that proposal has run in the migration script